src/rf24.c

```diff
@@ -50,21 +50,26 @@
  */
 #define OPERATING_FREQUENCY_WIDTH_MHZ 125U
 
+/**
+ * @brief Max number of pipes available.
+ */
+#define MAX_NUM_OF_PIPES 6
+
 /*****************************************
  * Private Macros
  *****************************************/
 
-#define _BS(bit) (1 << (bit))
+#define BIT_SHIFT_LEFT(bits) (1 << (bits))
```

Comment:
Eu acho `1 << coisa` direto no código 92389128 vezes melhor que colocar uma macro pra isso, já sabe o quanto se está shiftando, além disso, se for manter uma macro, deveria ser _BV(x), que coresponde a "bit value" e é mais "standard" em libs e dá pra saber do que se trata de imediato

Reply:
Acho que pode ser